### DISCUSSION TITLE
Possible to disable bitcode for Debug and Release?

### DISCUSSION THREAD

**Discussion by dillonhafer** [expo/expo#18478] [Archived: Questions]
The bitcode config setting in app.json only allows for Debug or Release, but in my case I need bitcode disabled in all schemes. Is there a way to configure app.json to use something like "none"? Or do I need to just change to using a `app.config.js` file now and dynamically switch to the opposite scheme so that it is always disabled?

**Comment by ngima** [comment]
You can disable bitcode with custom Plugin.

```javascript
//./plugins/withNoBitcode.js
const { withXcodeProject } = require('@expo/config-plugins');


module.exports = function withNoBitcode(config) {
    return withXcodeProject(config, async (config) => {

        const xcodeProject = config.modResults;
        xcodeProject.addBuildProperty('ENABLE_BITCODE', 'NO');

        return config;
    });
}
```


```javascript
{
    expo: {
        plugins: [
            "'./plugins/withNoBitcode'"
        ]
    }
}
```
